cmd/influxd/main.go

```diff
@@ -87,27 +88,27 @@ func (m *Main) Run(args ...string) error {
 
 		signalCh := make(chan os.Signal, 1)
 		signal.Notify(signalCh, os.Interrupt, syscall.SIGTERM)
-		m.Logger.Println("Listening for signals")
+		log.Info("Listening for signals")
 
 		// Block until one of the signals above is received
 		select {
 		case <-signalCh:
-			m.Logger.Println("Signal received, initializing clean shutdown...")
+			log.Info("Signal received, initializing clean shutdown...")
 			go func() {
 				cmd.Close()
 			}()
 		}
 
 		// Block again until another signal is received, a shutdown timeout elapses,
 		// or the Command is gracefully closed
-		m.Logger.Println("Waiting for clean shutdown...")
+		log.Info("Waiting for clean shutdown...")
 		select {
 		case <-signalCh:
-			m.Logger.Println("second signal received, initializing hard shutdown")
+			log.Warn("second signal received, initializing hard shutdown")
```

Comment:
I know we don't particularly want to use warn very much, but I do think this is a reasonable place to use it. A second signal could end up getting received by an automated system and it would be important information to know that, say, systemd needed to use a hard shutdown.